src/main/java/tds/exam/results/validation/impl/XSDBackedTDSReportValidator.java

```diff
@@ -25,13 +27,16 @@
     private final ExamResultsTransmitterServiceProperties properties;
 
     XSDBackedTDSReportValidator(final JAXBContext jaxbContext,
-                                @Value("classpath:xsd/TestResultsTransmissionFormat_Schema.xsd") final Resource xsd,
+                                @Value("classpath:/xsd/TestResultsTransmissionFormat_Schema.xsd") final Resource xsd,
                                 final ExamResultsTransmitterServiceProperties properties) throws IOException, SAXException {
         this.jaxbContext = jaxbContext;
         this.properties = properties;
-        SchemaFactory sf = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI);
-        Schema schema = sf.newSchema(xsd.getFile());
-        validator = schema.newValidator();
+        final SchemaFactory sf = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI);
+
+        try (final InputStream xsdStream = xsd.getInputStream()) {
+            final Schema schema = sf.newSchema(new StreamSource(xsdStream));
+            validator = schema.newValidator();
+        }
```

Comment:
It's not something I would have thought of.  Although I suppose the exceptions make sense once we started seeing them...